### ISSUE TITLE
map tag crash on steam version

### ISSUE THREAD

**Issue by daniel-tp** [L0laapk3/FactorioMaps#26]
Trying to run auto.py on [this map](https://my.mixtape.moe/bvskbg.zip) in Factorio 16.5.1 via Steam causes the following error to occur:

```
Traceback (most recent call last):  
  File "auto.py", line 559, in <module>  
    auto(*sys.argv[1:])  
  File "auto.py", line 464, in auto  
    rawPath = rawTags[tag["iconType"] + tag["iconName"][0].upper() + tag["iconName"][1:]]  
KeyError: 'itemIron-plate'
```

The map only had QoL mods and nothing that added new items, still tried disabling all mods and trying again, no change.

Full run log:

```
python auto.py --dayonly --hd
No save name passed. Using most recent save: trunklunk
factorio path: C:\Program Files (x86)\Steam\steamapps\common\Factorio\bin\x64\factorio.exe
output folder: script-output\FactorioMaps\trunklunk
checking for updates
enabling FactorioMaps mod
cleaning up
building autorun.lua
starting factorio
WARNING: running in limited support mode trough steam. Consider using standalone factorio instead.
         Please open steam and confirm the steam 'start game with arguments' popup.
Processing trunklunk/45/nauvis/day (1 of 1)
crop 100.0% [=========================================================================================================]
killed factorio
zoom 100.0% [=========================================================================================================]
generating mapInfo.json
updating labels
disabling FactorioMaps mod
cleaning up
Traceback (most recent call last):
  File "auto.py", line 559, in <module>
    auto(*sys.argv[1:])
  File "auto.py", line 464, in auto
    rawPath = rawTags[tag["iconType"] + tag["iconName"][0].upper() + tag["iconName"][1:]]
KeyError: 'itemIron-plate'
```

**Comment by L0laapk3** [comment]
This is a bug in the code I have right now, you can avoid it by using the standalone factorio from https://factorio.com/ , or by waiting one version until I fix it. Thanks for reporting!

**Comment by L0laapk3** [comment]
Fixed as of v3.2.1
